docs/configure-archive-logs.md

```diff
@@ -59,3 +60,14 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Why doesn't Argo Workflows recommend using this feature?
+
+Argo Workflows log storage facilities are quite basic. It is recommended that you use a combination of:
+
+* A fully featured Kubernetes capable logging facility which will provide you with facilities for indexing, searching and managing of log storage.
+* Use [links](links.md) to connect from the Argo Workflows user interface to your logging facility
+    * Use the `scope: workflow` link to get all the logs for a workflow, using the workflow name in the link `${metadata.name}` and the namespace `${metadata.namespace}`
+    * Use `scope: pod-logs` for those from a specific pod of name `${metadata.name}`
+
+There is no intention to substantially improve the logging facilities provided by Argo Workflows, this is considered best implemented in a separate product.
```

Comment:
Hmmm I don't think we need to state this directly, that's why we have a big warning and other recommendations after all.

Also I did want to investigate potentially having an `iframe` or some other more direct integration with a logging facility inside the Argo UI, similar to Grafana's more recent logs integration. I might consider that to be a "substantial improvement".

```suggestion
```

If you want to keep this in, I would make it more specific and change up the wording:
```suggestion
We do not plan to substantially improve Archive Logging as there are more optimized, purpose-built logging tools available in the ecosystem.
```

- use active voice, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)
- Argo is implied, since this is Argo's docs. Instead let's refer to the specific feature
  - "logging facilities" could refer to the live logs as well, which may still have features
- "tool" vs. "product"
- more specific as to why

Reply:
I've left something in to stop people expecting the kind of improvements we'd reject and rejigged it to put the suggested alternatives as part of this.

Reply:
Small follow-up [below](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1483667438)